src/main/kotlin/com/cognifide/gradle/aem/environment/tasks/EnvironmentHosts.kt

```diff
@@ -1,21 +1,28 @@
 package com.cognifide.gradle.aem.environment.tasks
 
 import com.cognifide.gradle.aem.AemDefaultTask
-import com.cognifide.gradle.aem.environment.hosts.Hosts
 import org.gradle.api.tasks.TaskAction
+import org.gradle.internal.os.OperatingSystem
 
 open class EnvironmentHosts : AemDefaultTask() {
 
     init {
-        description = "Append environment hosts to system hosts file. Requires super/admin user privileges."
+        description = "Prints environment hosts entries."
+    }
+
+    private val hostsFile = when {
+        OperatingSystem.current().isWindows -> """C:\Windows\System32\drivers\etc\hosts"""
+        else -> "/etc/hosts"
     }
 
     @TaskAction
     fun appendHosts() {
-        val hosts = Hosts.of(aem.environment.hosts.defined)
-        hosts.append()
-
-        aem.notifier.notify("Environment hosts", "Appended with success")
+        logger.lifecycle("""Printing hosts entries (append them to $hostsFile):""")
+        logger.quiet("## Generated by Gradle AEM Environment Plugin - START")
+        aem.environment.hosts.defined.forEach {
```

Comment:
Log once concatenated text. I am not sure if on debug level all entries will be logged with separate timestamp as prefix.

Reply:
it is not meant to be used in debug mode to copy this entry. It is used as follows:
`./gradlew environmentHosts -q` - in iscripts
or
`./gradlew environmentHosts` - by the user

Reply:
I was writing more about 

`logger.quiet(aem.environment.hosts.defined.joinToString("\n") { it.text })`

Reply:
OK